scm/src/GFS_typedefs.F90

```diff
@@ -113,6 +116,7 @@ module GFS_typedefs
     real(kind=kind_phys) :: dt_phys              !< physics  time step in seconds
 !--- restart information
     logical :: restart                           !< flag whether this is a coldstart (.false.) or a warmstart/restart (.true.)
+    logical :: cycling                           !< flag whether this is a coldstart (.false.) or a cycled run (.true.)
```

Comment:
Note that this will be removed again in fv3atm in the near future, see https://github.com/NOAA-EMC/fv3atm/pull/140.

Reply:
OK, thanks. No one would (should) ever change that from the default false value when running the SCM anyway.